src/theme.ts

```diff
@@ -61,6 +61,15 @@ declare module '@mui/material/styles/createPalette' {
 }
 
 const defaultTheme = {
+  breakpoints: {
+    values: {
+      xs: 0,
+      sm: 600,
+      md: 900,
+      lg: 1200,
+      xl: 1400, // custom xl to transition background on Home page
```

Comment:
Change looks harmless... just curious - any cons to 1400 vs 1536?

Reply:
not that I know of, unless we're breaking any other styles/layouts on `xl` then this shouldn't affect anything